pkg/apis/policy/validation/validation.go

```diff
@@ -17,13 +17,22 @@ limitations under the License.
 package validation
 
 import (
+	"fmt"
+	"path/filepath"
 	"reflect"
+	"strings"
 
 	unversionedvalidation "k8s.io/apimachinery/pkg/apis/meta/v1/validation"
+	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apimachinery/pkg/util/validation/field"
+	api "k8s.io/kubernetes/pkg/apis/core"
```

Comment:
please use `core` as alias instead. We want to get rid of the legacy `api`.

Reply:
Thanks! I updated the code to cover your comments.